packages/tuf/src/extraction.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+const START = `"signed":{`
+
+/**
+ * Extract the raw value under "signed" from a JSON string
+ * the JSON string MUST be canonical, especially there MUST NOT be any spaces between literals
+ * @param {string} str
+ */
+function extractSigned (str) {
```

Comment:
I don't see any unit tests for this function?

This is nitpicking because it is tested indirectly through `packages/tuf/test/integrity.spec.js`

Reply:
Yes! I still think I should add some to test edge cases but the first lines of the function should give us goo stability checks